lib/diagnostics_channel.js

```diff
@@ -0,0 +1,141 @@
+'use strict';
+
+const {
+  ObjectDefineProperty,
```

Comment:
avoid calling mutable global (see below)
```suggestion
  ArrayPrototypePush,
  ObjectDefineProperty,
```